### ISSUE TITLE
Back Up Team on Both Alliances

### ISSUE THREAD

**Issue by WBAucoin** [arthurlockman/gatool-ui#239]
This would require human error, but you can assign a backup team to both alliances.
![image](https://github.com/arthurlockman/gatool-ui/assets/139565335/4d710fb4-b0c4-40b4-a2dd-32d5672fdbfc)

**Comment by jameslockman** [comment]
Yes, this is true. We did not build logic to exclude previously assigned teams. We can investigate the level of effort.

**Comment by WBAucoin** [comment]
I would not fret over it. Much more important fish to fry.
